package.json

```diff
@@ -25,7 +25,7 @@
     "core-object": "^1.1.0",
     "ember-cli": "0.2.7",
     "ember-cli-app-version": "0.3.3",
-    "ember-cli-aptible-shared": "0.0.24",
+    "ember-cli-aptible-shared": "0.0.25",
```

Comment:
I just updated https://github.com/aptible/dashboard.aptible.com/pull/370 to fix the hanging test issue.  I'll merge once green, and then you should be able to rebase this...